### PR TITLE
add retry times for user

### PR DESCRIPTION
**Pull request by wenxuwan**
**What this PR does**:
add retry times for user

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

**Review by Patrick0308 — APPROVED** [review]
Pls resolve the hound's comment.

**Comment by wenxuwan** [comment]
> Pls resolve the hound's comment.

done

File: zk/server_help.go [hunk 1]
```diff
@@ -39,6 +54,10 @@ func StartTestCluster(size int, stdout, stderr io.Writer) (*TestCluster, error)
 			cluster.Stop()
 		}
 	}()
+	options := &options{retryTimes: 10}
```
Comment (LaurenceLiZhixin) [inline zk/server_help.go:57]: Why not use WithRetryTimes() func to init

Comment (wenxuwan) [reply zk/server_help.go:57]: default value, show declared variables is ok

**Review by LaurenceLiZhixin — APPROVED** [review]

File: zk/server_help.go (hunk 1 above)
Comment (LaurenceLiZhixin) [reply zk/server_help.go:57]: Fine ^_^

**Pull request merged**